javabuilder-authorizer/websocket_authorizer_function.rb

```diff
@@ -81,3 +74,22 @@ def get_token_info(context, sid)
 def get_region(context)
   context.invoked_function_arn.split(':')[3]
 end
+
+def error(client, status, function_name, sid)
+  puts "TOKEN VALIDATION ERROR: #{status} token_id: #{sid}"
+  metric_data = {
```

Comment:
Hmm yeah, it might be nice to pull this out into a helper to share between the two authorizers? Maybe a `MetricsReporter(/Helper)` with a `report_token_error(status, function_name)`? That helper module could also own the cloudwatch client then?

Reply:
Yeah that makes sense, I think I was getting bogged down by the fact that I think the dynamodb stuff in websocket_authorizer should also be moved into a helper. Lemme see if I can at least share the cloudwatch code between the two.